src/apis/meeting/api.ts

```diff
@@ -25,8 +25,10 @@ export const meeting = {
   /**
    * @description 모임 일정 시간 조회
    */
-  times: async (uuid: string) => {
-    const response = await instance.get<TimesResponse>(`/meetings/${uuid}/times`);
+  times: async (uuid: string, sort?: string) => {
+    const response = await instance.get<TimesResponse>(
+      `/meetings/${uuid}/times?sort=${sort === 'LATEST' ? 'LATEST' : ''}`
+    );
```

Comment:
타입으로 sort 옵션을 안내할 수 있으면 좋을 것 같아용

```suggestion
  times: async (uuid: string, sort?: 'LATEST' | 'COUNT') => {
    const response = await instance.get<TimesResponse>(
      `/meetings/${uuid}/times?sort=${sort ?? 'COUNT'}`
    );
```